cookbook/security/voters_data_permission.rst

```diff
@@ -213,9 +212,7 @@ from the authorization checker is called.
             $post = ...;
 
             // keep in mind, this will call all registered security voters
-            if (false === $this->get('security.authorization_checker')->isGranted('view', $post)) {
-                throw new AccessDeniedException('Unauthorised access!');
-            }
+            $this->denyAccessUnlessGranted('view', $post, 'Unauthorised access!');
```

Comment:
When using shortcuts, we usually show the long/boring alternative in commented code:

```php
    $post = ...;

    // keep in mind, this will call all registered security voters
    $this->denyAccessUnlessGranted('view', $post, 'Unauthorised access!');

    // the equivalent code without using the denyAccessUnlessGranted() shortcut
    // use Symfony\Component\Security\Core\Exception\AccessDeniedException;
    //
    // if (false === $this->get('security.authorization_checker')->isGranted('view', $post)) {
    //     throw new AccessDeniedException('Unauthorised access!');
    // }
```